content/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -263,13 +264,26 @@ click on a test file named `spec-a.js` via
 [cypress open](/guides/guides/command-line#cypress-open), then the Cypress App
 executes the files in the following order:
 
-```html
-<!-- bundled support file -->
-<script src="support/index.js"></script>
-<!-- bundled spec file -->
-<script src="integration/spec-a.js"></script>
+:::e2e-component-example
+
+```js
+//bundled e2e support file
+<script src="support/e2e.js"></script>
+
+//bundled spec file
```

Comment:
It turns out we're no longer bundling spec files, so can you update this content to reflect that? It could be as simple as changing the word "bundled" to "your" when referring to spec files in this section.

Reply:
What do you think of this? https://github.com/cypress-io/cypress-documentation/pull/4328/commits/54add477b1b893981187316783abde8162a64c6e

Reply:
https://github.com/cypress-io/cypress-documentation/pull/4328/commits/54add477b1b893981187316783abde8162a64c6e